13/umbraco-deploy/deploy-settings.md

```diff
@@ -309,13 +311,7 @@ For Umbraco 10, by default, a SQLite database is created.
 If you would prefer to use SQL Server LocalDb when it's available on your local machine, set this value to `true`. If LocalDB isn't reported as being available by Umbraco, it will fallback to using a SQLite database instead.
 
 ```json
-    "Umbraco": {
-        "Deploy": {
-            "Settings": {
-                "PreferLocalDbConnectionString": true
-            }
-        }
-    }
+"PreferLocalDbConnectionString": true
```

Comment:
should this be changed only for v13 or for all versions?

Reply:
This can be updated for all versions: other code examples also don't show the `Umbraco:Deploy:Settings` part (as that's already mentioned at the top of the article)...